derive/src/derive_struct.rs

```diff
@@ -21,12 +21,21 @@ impl DeriveStruct {
             .with_return_type("core::result::Result<(), bincode::error::EncodeError>")
             .body(|fn_body| {
                 for field in fields.names() {
-                    fn_body
-                        .push_parsed(format!(
-                            "bincode::enc::Encode::encode(&self.{}, &mut encoder)?;",
-                            field.to_string()
-                        ))
-                        .unwrap();
+                    if field.has_field_attribute(FieldAttribute::WithSerde) {
+                        fn_body
+                            .push_parsed(format!(
+                                "bincode::__serde_encode_field(&self.{}, &mut encoder)?;",
```

Comment:
This would work for `Encode` and `Decode`, but not for `BorrowDecode`.

I think I'll make a wrapper `bincode::serde::Compat` (Decode/Encode) and `bincode::serde::BorrowCompat` (BorrowDecode/Encode)

Reply:
Done!